test/workflows/app.yaml

```diff
@@ -1,4 +1,4 @@
-apiVersion: 0.1.0
+apiVersion: 0.3.0
 environments:
   releasing:
     destination:
```

Comment:
Should we also try to upgrade version of GCP cluster to 1.12 here: https://github.com/kubeflow/katib/blob/master/test/scripts/v1alpha2/create-cluster.sh#L35 and
https://github.com/kubeflow/katib/blob/master/test/scripts/v1alpha1/create-cluster.sh#L35?

I think, in Kubeflow they are using 1.12 now: https://github.com/kubeflow/kubeflow/pull/3440